RealmSwift/List.swift

```diff
@@ -686,3 +686,33 @@ extension List: PropertyNameConvertible {
         return (key: rlmArray.propertyKey, isLegacy: rlmArray.isLegacyProperty)
     }
 }
+
+// MARK: Aggregatable
+
+extension List: Aggregatable where Element: ObjectBase {
+    public func min<T>(ofProperty property: KeyPath<Element, T>) -> T? where T: MinMaxType {
+        self.min(ofProperty: _name(for: property))
+    }
+
+    public func max<T>(ofProperty property: KeyPath<Element, T>) -> T? where T: MinMaxType {
+        self.max(ofProperty: _name(for: property))
+    }
+
+    public func sum<T>(ofProperty property: KeyPath<Element, T>) -> T where T: AddableType {
+        self.sum(ofProperty: _name(for: property))
+    }
+
+    public func average<T>(ofProperty property: KeyPath<Element, T>) -> T? where T: AddableType {
+        self.average(ofProperty: _name(for: property))
+    }
```

Comment:
These look like they should be implementable as a default protocol extension rather than on each type.